subworkflows/local/prepare_genome/main.nf

```diff
@@ -55,7 +55,7 @@ workflow PREPARE_GENOME {
                 ch_bowtie_index = UNTAR_BOWTIE_INDEX.out.files
                 ch_versions  = ch_versions.mix(UNTAR_BOWTIE_INDEX.out.versions)
             } else {
-                ch_bowtie_index = Channel.fromPath("${val_bowtie_index}**ebwt", checkIfExists: true).map{it -> [ [id:it.baseName], it ] }.collect()
+                ch_bowtie_index = Channel.fromPath("${val_bowtie_index}**ebwt", checkIfExists: true).map{it -> [ [id:'bowtie_index'], it ] }.collect()
```

Comment:
`Mirdeep2` is a bit tricky in how the index should be named: https://github.com/nf-core/smrnaseq/blob/adc43dbbe5ce454939aeb7e3717b24d75b44e5c8/modules/nf-core/mirdeep2/mapper/main.nf#L30.

So if the path to the index `ch_bowtie_index` is something like:
`/work/bowtie_index`
and the files inside that directory are:
`/work/bowtie_index/genome.1.ebwt`
`/work/bowtie_index/genome.2.ebwt`
`/work/bowtie_index/genome.3.ebwt`

Then the meta should be: `genome`.

This renaming is done automatically when indexing the fasta in the else statement:


```
            // Index FASTA with nf-core Bowtie1
            INDEX_GENOME ( CLEAN_FASTA.out.output )
            ch_versions      = ch_versions.mix(INDEX_GENOME.out.versions)
```

Reply:
Also to be addressed in https://github.com/nf-core/smrnaseq/issues/451